pkg/services/install/install.go

```diff
@@ -0,0 +1,150 @@
+package install
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/weaveworks/weave-gitops/pkg/models"
```

Comment:
NIt: I find the code nicer when the imports are grouped together alphabetically. i.e. remove the whitespace between these imports.